lib/DoorControl/Controller/DoorControl.pm

```diff
@@ -15,6 +17,7 @@ sub unlock {
     my $name = $self->param('name');
     my $pin = $self->param('pin');
     my $remember = $self->param('remember');
+    my $ip = $self->param('ip');
```

Comment:
This parameter was never created but never implemented into the program so it needs to be removed